fusor-ember-cli/app/controllers/openstack/assign-nodes.js

```diff
@@ -204,10 +200,6 @@ const AssignNodesController =  Ember.Controller.extend(DeploymentControllerMixin
       this.closeEditDialog();
     },
 
-    cancelEditRole() {
-      this.closeEditDialog();
```

Comment:
I just closed the modal in the component rather than passing this action to the component.